recipes/openscenegraph/all/conanfile.py

```diff
@@ -0,0 +1,594 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ConanFile(ConanFile):
+    name = "openscenegraph"
+    description = "OpenSceneGraph is an open source high performance 3D graphics toolkit"
+    topics = "conan", "openscenegraph", "graphics"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.openscenegraph.org"
+    license = "LGPL-2.1-only", "WxWindows-exception-3.1"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_applications": [True, False],
+        "build_examples": [True, False],
+        "enable_notify": [True, False],
+        "enable_deprecated_api": [True, False],
+        "enable_readfile": [True, False],
+        "enable_ref_ptr_implicit_output_conversion": [True, False],
+        "enable_ref_ptr_safe_dereference": [True, False],
+        "enable_envvar_support": [True, False],
+        "enable_windowing_system": [True, False],
+        "enable_deprecated_serializers": [True, False],
+        "use_fontconfig": [True, False],
+        # "with_asio": [True, False], # osg seems to not work with recent versions of asio
+        # "with_collada": [True, False],
+        "with_curl": [True, False],
+        # "with_dcmtk": [True, False], # requires the usage of cmake_find_package, which is broken due to #2311
```

Comment:
What is broken exactly in `cmake_find_package` regarding dcmtk?

Reply:
dcmtk works fine from my testing on linux when using `cmake_find_package`, the problem is that the opengl package generates a find package script that isn't compatible with the one that is shipped as a part of cmake on case-insensitive filesystems. This means that any cmake based library that uses opengl cannot currently use the `cmake_find_package` generator. See #2311.

Reply:
Therefore why not using `cmake_find_package_multi` to generate config files instead of module files? No conflict with official module file because module file are checked before config files.

Reply:
I just pushed a commit to do this. It seems to work for most packages, although it fails specifically for dcmtk. No real idea why.